### ISSUE TITLE
/primary_care/CPRD_lipid.csv has wrong columns and missing data

### ISSUE THREAD

**Issue by robin-sun** [spiros/chronological-map-phenotypes#1]
In /primary_care/CPRD_lipid.csv, I see different column names.
"Disease","Read code","Medcode","Descr",**"Incl_Cat","In_CALIBER"**

Also, some of the category fields are left as "42", like
"dyslipidaemia","C321.12","99456.0","Very low density lipoprotinaemia","1.0","42"

**Comment by spiros** [comment]
Thank you for reporting this - will have a look and get back to you.

**Comment by robin-sun** [comment]
Also if you don't mind,  could you also review
CPRD_folatedef.csv
CPRD_leiomyoma.csv
These 2 files seem to have some missing data or extra data as well.

**Comment by spiros** [comment]
Thanks, noted, will check the other files as well and push updates & reply here.

**Comment by spiros** [comment]
Hey, thanks for reporting these. 

I've pushed a number of fixes hopefully addressing your comments:

1. Phenotypes for total cholesterol, raised LDL , raised triglycerides and low HDL do not make use of diagnosis codes (only biomarker information) so I've removed the CSV files to avoid confusion 3b96f6f67539e7b3fe3d6263749b70f0b7dd53a5 

2. The Read codes at the bottom of the CSV for the leiomyoma phenotype were diagnosis codes related to leiomyoma during pregnancy but were not used in the study so have removed these from the CSV 7386ff1ef323d26736571f196ff0603d56b8ed47

3. Updated a couple of files by removing "possible" diagnosis codes which were not used in the phenotypes 9c40d60e0eb381439e4155e8117641f4b4104ac6
